windows/flutter/generated_plugins.cmake

```diff
@@ -6,6 +6,9 @@ list(APPEND FLUTTER_PLUGIN_LIST
   url_launcher_windows
 )
```

Comment:
@guidezpl I don't understand why and how this file got generated, can you please guide me on this?

Reply:
pub get with latest flutter master , automatically adding it

Reply:
That's fine